src/descriptors.ts

```diff
@@ -20,6 +21,7 @@ export interface MultisigWalletConfig {
   addressType: MultisigAddressType;
   keyOrigins: KeyOrigin[];
   network: Network | "bitcoin";
+  useRangeNotation?: boolean;
 }
 
 export const decodeDescriptors = async (
```

Comment:
should we handle decode range encoded descriptor as well?

Reply:
I don't think that quite works. If you're trying to decode a range notation/multi-path descriptor, having two inputs for internal and external doesn't make any sense. I think if we're doing a separate function for encode, having a separate one for multipath would make sense too, e.g. 
```
export const decodeMultipathDescriptor = (descriptor: string, network?: Network | "bitcoin") => { ... } 
```